front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -66,6 +67,52 @@ const usePathfinding = ({
   const [postPathProperties] =
     osrdEditoastApi.endpoints.postInfraByInfraIdPathProperties.useLazyQuery();
 
+  const [postSearch] = osrdEditoastApi.endpoints.postSearch.useMutation();
+
+  const fetchOperationalPoints = useCallback(
+    async (steps: PathStep[]) => {
+      if (!infraId || !steps.length) return;
+
+      const searchPayload = buildOpSearchQuery(infraId, steps);
+
+      if (!searchPayload) return;
+      let results: SearchResultItemOperationalPoint[];
+      try {
+        results = (await postSearch({
+          searchPayload,
+          pageSize: 101,
+        }).unwrap()) as SearchResultItemOperationalPoint[];
+      } catch (error) {
+        console.error('Error fetching operational points:', error);
```

Comment:
We should `return` after printing the error, or else we `results` is left undefined and will cause a runtime error below when calling `results.find()`.

I'm actually surprised TypeScript doesn't complain about this.